JS/fluentRevealNavbar.uc.js

```diff
@@ -63,6 +84,13 @@
      * @param {object} e (event)
      */
     handleEvent(e) {
+      /// filter out mouse events which are too far from toolbar to cause any actual redraw
+      /// value is {gradientSize} + some additional padding to make sure effect fully clears out
+      if (this._options.filterDy && e.clientY > this._options.gradientSize + 25) {
```

Comment:
```suggestion
      if (
        this._options.filterDy &&
        e.clientY >
          document.getElementById("browser").getBoundingClientRect().y +
            this._options.gradientSize
      ) {
```

Reply:
Probably want to change the comments too since this changes the meaning of filterDy.
It's filtering out mouse events that are more than {gradientSize} pixels below the toolbar.

Reply:
The reasoning for this is that the toolbar height is variable, not fixed. For example, Ctrl+Shift+B makes the bookmarks toolbar appear, which significantly increases its height. There are also infobars that can rarely appear below the main toolbar, above the browser content viewer.